charts/grafana/values.yaml

```diff
@@ -73,7 +74,7 @@ livenessProbe:
 # schedulerName: "default-scheduler"
 
 image:
-  repository: grafana/grafana
+  repository: grafana/grafana-enterprise
```

Comment:
it might be breaking change for somebody. I know that some companies do not use docker hub directly, instead of this they use company's docker repository proxy and they use only some list of allowed docker images. 
so, if they do not have this image in the white list, their k8s cluster might be not able to download the image. 
I think we need to mention it in upgrade guide: https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md#upgrading-an-existing-release-to-a-new-major-version

Reply:
Also a very valid point.